pkgs/applications/science/robotics/betaflight-configurator/default.nix

```diff
@@ -0,0 +1,52 @@
+{stdenv, fetchurl, unzip, makeDesktopItem, nwjs }:
+
+stdenv.mkDerivation rec {
+  strippedName = "betaflight-configurator";
+  name = "${strippedName}-${version}";
+  version = "10.4.0";
+  src = fetchurl {
+    url = "https://github.com/betaflight/betaflight-configurator/releases/download/${version}/${strippedName}_${version}_linux64.zip";
+    sha256 = "0njj47cgjw8h0ani1hg6zw2iafps95mh9rx966ir842nvpprs9dc";
+  };
+
+  buildInputs = [ unzip ];
+
+  desktopItem = makeDesktopItem {
+    name = strippedName;
+    exec = strippedName;
+    icon = "${strippedName}-icon.png";
+    comment = "Betaflight configuration tool";
+    desktopName = "Betaflight Configurator";
+    genericName = "Flight controller configuration tool";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin \
+             $out/opt/${strippedName} \
+             $out/share/icons
+
+    cp -r . $out/opt/${strippedName}/
+
+    cat <<EOF > $out/bin/${strippedName}
```

Comment:
`writeShellScriptBin` might be useful for this

Reply:
The script itself must now the `$out` path of the `betaflight-configurator` package. How can I do that with `writeShellScriptBin`? I will update the PR with my current approach (though it's not working) to clarify the problem.

Reply:
@corngood  ping

Reply:
You could use `makeWrapper "${nwjs}/bin/nw" $out/bin/${strippedName} --add-flags "$out/opt/${strippedName}"`

Reply:
Did you try this ? It would simplify it a bit

Reply:
@nyanloutre Please pardon the late feedback. Indeed, `makeWrapper` works fine. Thank you for the suggestion!